main.js

```diff
@@ -0,0 +1,197 @@
+//global variables
+
+let board = document.querySelectorAll('div[id^="cell-"]');
+let startButton = document.getElementById("start");
+let statusArea = document.getElementById("statusLine");
+let resetButton = document.getElementById('reset')
+
+// game class
+class Game {
+  constructor() {
+    this.turn = false;
+    this.message = {
+      playerXTurn: "Player X's turn",
+      playerOTurn: "Player O's turn",
+    };
+  }
+  playerTurn = () => {
+    this.turn = !this.turn;
+    if (this.turn) {
+      return "playerXTurn";
+    } else {
+      return "playerOTurn";
+    }
+  }
+  printMessages = (gameStatus) => {
+    statusArea.textContent = this.message[gameStatus];
+  }
+  //
+  drawLetter = (turn) => {
+    if (turn ==="playerOTurn") {
+      return "X";
+    } else {
+      return "O";
+    }
+  };
+}
+
+//Creates array of the board in current state
+let boardArr = []
+  let boardArray = () => {
+    for (let i = 0; i < board.length; i++) {
+      boardArr[i] = board[i].textContent
+    }
+    return boardArr
+
+  }
+  
+  //breaks array into 3 sub-arrays in order to be checked for an across or vertical win
+  breakIntoSubArrays = (anotherArray)=> {
+    boardSubArray1 = anotherArray.slice(0,3)
+    boardSubArray2 = anotherArray.slice(3,6)
+    boardSubArray3 = anotherArray.slice(6,9)
+    rebuiltArray = [boardSubArray1,boardSubArray2,boardSubArray3]
+    return rebuiltArray
+  }
+
+//checks wins vertically and horizontally
+checkWinUpDownAndAcross = (someArray) => {
+  for(var i = 0; i<3;i++) {
+    var colSum = "";
+    for(var j = 0; j<3;j++){
+        colSum += someArray[j][i];
+    }
+    if (colSum === ("OOO")) {
+    cancelEventListeners()
+    statusArea.textContent = "Circle's Win"
+    return true
+  } else if (colSum === "XXX") {
+    cancelEventListeners()
+    statusArea.textContent = "Cross's Win";
+    return true
+    }
+  }
+    
+  for(var i = 0; i<3;i++){
+      var rowSum = "";
+      for(var j = 0; j<3;j++){
+          rowSum += someArray[i][j];
+      }
+      if (rowSum === ("OOO")) {
```

Comment:
This is a great way to check for a win. You could even probably use this code to check the diagonals  as well if you had an array of arrays for win conditions, or separate arrays for each win condition.